code/control/solution/control.sol.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+
+constexpr int numbers[]{1, 2, 3, 4, 5, 6, 7, 8, 9};
+
+bool isodd(int i) { return i % 2; }
+
+void part1() {
+  int sum_odd = 0;
+  int sum_eve = 0;
+  for (int num : numbers) {
+    (isodd(num) ? sum_odd : sum_eve) += num;
```

Comment:
Do you consider this good code? I have seen this seldomly. But it's neat somehow :)

Reply:
I was wondering. The alternative duplicates `+= num`, so has also small defect. Don't know honestly. Mayeb duplication is more clear here as it's for beginners ? Maybe also this teaches more to them ? What do other think ?

Reply:
After a quick discussion in the meeting, I left it, with an extra explanatory comment